### ISSUE TITLE
How to validate

### ISSUE THREAD

**Issue by pehkawn** [JuliaAcademy/JuliaTutorials#80]
Not sure if this is the right forum for this, but I have some issue with the last line in lesson 01. "Getting started". The line reads `Please click on `Validate` on the top, once you are done with the exercises.` However, I can not find any button, or any text for that matter, with this word. Am I doing something wrong, or is this line redundant?

**Comment by logankilpatrick** [comment]
No, you are not doing anything wrong. That line refers to an old Jupyter interface. I will try to hunt down and remove those lines. Thanks for the heads up!
